pandas/core/frame.py

```diff
@@ -7656,7 +7658,13 @@ def transform(
         return result
 
     def apply(
-        self, func, axis: Axis = 0, raw: bool = False, result_type=None, args=(), **kwds
+        self,
+        func: ApplyFuncType,
```

Comment:
Just to make it explicit, this annotation will show up as `Union[np.ufunc, Union[Callable, str, List[Union[Callable, str]], Dict[Union[Hashable, NoneType], Union[Callable, str, List[Union[Callable, str]]]]]]`

(but so which is not even complete yet, since the ufunc should be present in each nested case Union[Callable, str]. But isn't a ufunc a callable, so that can maybe be removed?)

Reply:
Going off the aggregation case we don't special case `ufunc` there so we probably shouldn't do it here either.

In that case I __think__ `ApplyFuncType` is the same as `AggFuncType` so we can use the same alias in both cases

Reply:
yeah i agree here, no need to be different, let's use `AggFuncType`

Reply:
Done.